ghc/plan.sh

```diff
@@ -1,18 +1,28 @@
 pkg_name=ghc
 pkg_origin=core
-pkg_version=8.4.3
+pkg_version=8.6.3
 pkg_license=('BSD-3-Clause')
 pkg_upstream_url="https://www.haskell.org/ghc/"
 pkg_description="The Glasgow Haskell Compiler"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source="http://downloads.haskell.org/~ghc/${pkg_version}/ghc-${pkg_version}-src.tar.xz"
-pkg_shasum="ae47afda985830de8811243255aa3744dfb9207cb980af74393298b2b62160d6"
+pkg_shasum="9f9e37b7971935d88ba80426c36af14b1e0b3ec1d9c860f44a4391771bc07f23"
 
 pkg_bin_dirs=(bin)
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(lib/ghc-${pkg_version}/include)
 pkg_interpreters=(bin/runhaskell bin/runghc)
 
+pkg_build_deps=(
+  core/coreutils
+  core/binutils
+  core/diffutils
+  core/ghc82
```

Comment:
For consistency in the build chain, we'll want to use ghc84 to build ghc 8.6

Reply:
Should be resolved now :)